pkg/apps/blackduck/blackduck.go

```diff
@@ -1932,8 +1947,11 @@ func (b Blackduck) GetComponents(bd *blackduckapi.Blackduck, compType string) (*
 		return nil, fmt.Errorf("version %s is not supported", bd.Spec.Version)
 	}
 
+	log.Infof("bd: %+v", bd)
+
 	cp, err := store.GetComponents(version, b.config, b.kubeClient, bd)
 	if err != nil {
+		log.Errorf("cp: %+v, err: %+v", cp, err)
```

Comment:
No need to print the error here since we return it.

Reply:
fixed